fmriprep/cli/parser.py

```diff
@@ -315,6 +315,14 @@ def _slice_time_ref(value, parser):
 https://fmriprep.readthedocs.io/en/%s/spaces.html"""
         % (currentv.base_version if is_release else "latest"),
     )
+    g_conf.add_argument(
+        "--output-echos",
```

Comment:
```suggestion
        "--me-output-echos",
```

Signal the subtype of workflow like the `--fs-*`